.github/workflows/ci.yml

```
@@ -1,11 +1,11 @@
 # This is displayed in the pr check area
-name: Comparison to 5.X.X
+name: Comparison to publishable branch
 
 # Controls when the action will run. Triggers the workflow on pull request
-# events but only for the 5.X.X branch
+# events but only for the 5.X.X or 6.X.X branch
 on:
     pull_request:
-        branches: [5.X.X]
+        branches: [5.X.X, 6.X.X]
```

Kannst auch gerne einfach nur 6.X.X machen
```suggestion
        branches: [6.X.X]
```